client/cypress/integration/dashboard/dashboard_spec.js

```diff
@@ -342,20 +310,20 @@ describe('Dashboard', () => {
         });
 
         it('stays put when dragged under snap threshold', () => {
-          dragBy(cy.get('@textboxEl'), 90).then((delta) => {
-            expect(delta.left).to.eq(0);
+          dragBy(cy.get('@textboxEl'), 90).should(($el) => {
+            expect($el.offset().left).to.eq(15); // no change, 15 -> 15
           });
         });
 
         it('moves one column when dragged over snap threshold', () => {
-          dragBy(cy.get('@textboxEl'), 110).then((delta) => {
-            expect(delta.left).to.eq(200);
+          dragBy(cy.get('@textboxEl'), 110).should(($el) => {
+            expect($el.offset().left).to.eq(215); //  moved by 200, 15 -> 215
           });
         });
 
         it('moves two columns when dragged over snap threshold', () => {
-          dragBy(cy.get('@textboxEl'), 330).then((delta) => {
-            expect(delta.left).to.eq(400);
+          dragBy(cy.get('@textboxEl'), 330).should(($el) => {
```

Comment:
Here it's also possible to have Cypress' syntax :)
```javascript
dragBy(cy.get('@textboxEl'), 330)
  .invoke('offset')
  .its('left')
  .should('eq', 415); // moved by 400, 15 -> 415
```

Reply:
Actually, this won't work cause the `invoke+its` would execute only once (which would surely yield a fail), while the `except` within the `should` re-executes till passed or timed-out.

Reply:
I had had a similar thought, but ever since this worked (at least to track the change) I started to believe more in Cypress' magic :slightly_smiling_face: :
```javascript
cy.document()
    .its('fonts')
    .invoke('check', '14px FontAwesome')
    .should('eq', true);
```

![dashboard_spec](https://user-images.githubusercontent.com/3356951/56460658-7b76f980-637c-11e9-8b8f-00c4fa00c100.gif)

Reply:
I tried it locally and always get a fail 🤔
I need to watch your video slowed down. Can you send me the vid?

Reply:
That's weird :thinking: 

Another way to do it that is even safer as the command that precedes `should` is what should be repeated:
```javascript
dragBy(cy.get('@textboxEl'), 330)
  .invoke('offset')
  .should('have.property', 'left', 415); // moved by 400, 15 -> 415
```

> Can you send me the vid?

This one was created as a gif haha, I pushed to a temporary repo so we could test it over CI -> https://circleci.com/gh/getredash/redash/20305 (Succeeds)

Reply:
This works well 👍Updated.
```js
.invoke('offset')
.should('have.property', 'left', val);
```